governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
```

Comment:
```suggestion
It also has a roadmap for additions coming in the next iteration of governance (Governance 1.1).
```

Reply:
Just in case this doesn't end up as planned because of cross-commits I made here, I'm reopening this discussion so it appears when looking after the merge is complete.